server/controllers/index.js

```diff
@@ -0,0 +1,224 @@
+//module to contain the controller functions for postgres and cassandra
+
+//import the client and cassClient from server/index.js
+// const { pgClient, cassClient } = require('../index.js');
+
+
+//Require postgres
+const { Pool } = require('pg');
+
+//Create connection to postgres db
+const pgClient = new Pool({
+  user: 'jasonschreiber',
+  host: 'localhost',
+  database: 'titleservice',
+  password: 'password',
```

Comment:
This is probably fine for now, but I assume the connection details should be put in a .env file at some point